### PR TITLE
PK64RGB Keyboard Addition

### PR DESCRIPTION
**Pull request by yj7272098**
Add PK64RGB keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/playkbtw/pk64rgb/config.h [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#define DISABLE_RGB_MATRIX_BAND_VAL
+#define DISABLE_RGB_MATRIX_BAND_PINWHEEL_SAT
+#define DISABLE_RGB_MATRIX_BAND_PINWHEEL_VAL
+#define DISABLE_RGB_MATRIX_BAND_SPIRAL_SAT
+#define DISABLE_RGB_MATRIX_BAND_SPIRAL_VAL
+#endif
+
+#define RGB_DI_PIN B0
+#ifdef RGB_DI_PIN
+#    define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/playkbtw/pk64rgb/config.h:77]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/playkbtw/pk64rgb/rules.mk [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
```
Comment (drashna) [inline keyboards/playkbtw/pk64rgb/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/playkbtw/pk64rgb/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+![PK64RGB](https://cdn.store-assets.com/s/409567/i/25281841.png)
+
+A 60% RGB hotswap keyboard PCB made by Play Keyboard.
+
+* Keyboard Maintainer: [Barry Huang](https://github.com/yj7272098)
+* Hardware Supported: Play Keyboard 64 RGB
+* Hardware Availability: [Play Keyboard](http://play-keyboard.store/)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/playkbtw/pk64rgb/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/playkbtw/pk64rgb/rules.mk (hunk 2 above)
Comment (yj7272098) [reply keyboards/playkbtw/pk64rgb/rules.mk:10]: The firmware will be too large after modifying it into full bootmagic functions. Do I need to make this change?

Comment (sigprof) [reply keyboards/playkbtw/pk64rgb/rules.mk:10]: Full bootmagic is already removed, and `BOOTMAGIC_ENABLE = yes` actually enables Bootmagic Lite; the `lite` value is deprecated and scheduled to be removed.

https://docs.qmk.fm/#/ChangeLog/20210828?id=bootmagic-full-removal

Comment (yj7272098) [reply keyboards/playkbtw/pk64rgb/rules.mk:10]: > Full bootmagic is already removed, and `BOOTMAGIC_ENABLE = yes` actually enables Bootmagic Lite; the `lite` value is deprecated and scheduled to be removed.
> 
> https://docs.qmk.fm/#/ChangeLog/20210828?id=bootmagic-full-removal

I see. Thanks for clarifying. :)

**Comment by yj7272098** [comment]
Can someone help me progress this PR? Thanks. :)

**Review by fauxpark — APPROVED** [review]

**Comment by yj7272098** [comment]
Thanks!

**Pull request merged**